From bb31ba937f56aca1fc18e29873de8e7b73100c69 Mon Sep 17 00:00:00 2001
From: medusalix <ss7@live.de>
Date: Wed, 21 Feb 2024 21:02:37 +0100
Subject: [PATCH] hwdb: Add headset form-factor override for Xbox Wireless
 Dongle

The Xbox Wireless Headset does not get assigned a sound form factor
when connected via the dongle using the `xone` driver.
Add USB IDs for the old & new dongle and built-in variants.

(cherry picked from commit ca4b6b269d00c95bb52bbb57adf86776dc844a51)

Related: RHEL-5950
---
 hwdb.d/70-sound-card.hwdb | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/hwdb.d/70-sound-card.hwdb b/hwdb.d/70-sound-card.hwdb
index dd5ef05583..a7ea8c64ee 100644
--- a/hwdb.d/70-sound-card.hwdb
+++ b/hwdb.d/70-sound-card.hwdb
@@ -15,6 +15,16 @@
 usb:v1B1Cp0A51*
  SOUND_FORM_FACTOR=headset
 
+###########################################################
+# Microsoft
+###########################################################
+# Xbox Wireless Dongle
+usb:v045Ep02E6*
+usb:v045Ep02FE*
+usb:v045Ep02F9*
+usb:v045Ep091E*
+ SOUND_FORM_FACTOR=headset
+
 ###########################################################
 # Steelseries
 ###########################################################
